config/initializers/sprockets.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+Sprockets.register_compressor "application/javascript", :terser, Terser::Compressor
```

Comment:
It looks like this isn't needed to register it manually.

Reply:
Removed